### ISSUE TITLE
 Failed to watch *v1alpha1.CNINode log messages

### ISSUE THREAD

**Issue by tulanian** [aws/amazon-vpc-cni-k8s#2574]
**What happened**:
Upgraded  to v1.15.0-eksbuild.2 using  Addon Update. Pods are up but now seeing these logs:

```
023-09-18T10:08:00.724060030Z E0918 10:08:00.723932      10 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.3/tools/cache/reflector.go:231: Failed to watch *v1alpha1.CNINode: unknown (get cninodes.vpcresources.k8s.aws)
2023-09-18T10:08:33.576510894Z E0918 10:08:33.576407      10 reflector.go:148] pkg/mod/k8s.io/client-go@v0.27.3/tools/cache/reflector.go:231: Failed to watch *v1alpha1.CNINode: unknown (get cninodes.vpcresources.k8s.aws)
```

**Attach logs**

**What you expected to happen**:
Not to see these log messages.

**How to reproduce it (as minimally and precisely as possible)**:
Upgrade to  v1.15.0-eksbuild.2 via the Addon Update mechanism.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.26
- CNI Version:  v1.15.0-eksbuild.2
- OS (e.g: `cat /etc/os-release`): Amazon Linux 2
- Kernel (e.g. `uname -a`): 5.10.186-179.751.amzn2.x86_64

**Comment by tulanian** [comment]
[eks_i-0749b1b4e0d6f932d_2023-09-18_1040-UTC_0.7.6.tar.gz](https://github.com/aws/amazon-vpc-cni-k8s/files/12646940/eks_i-0749b1b4e0d6f932d_2023-09-18_1040-UTC_0.7.6.tar.gz)

**Comment by jdn5126** [comment]
@tulanian this log message is harmless and was first reported via https://github.com/aws/amazon-vpc-cni-k8s/pull/2567. The TL;DR is that the Kubernetes client wants to cache `CNINode` objects, but we do not need or want them to be cached.

The long-term fix, which will ship in the next `v1.15.x` release, is https://github.com/aws/amazon-vpc-cni-k8s/pull/2570. If you want to stop the log, you can add `watch` permissions as https://github.com/aws/amazon-vpc-cni-k8s/pull/2567 did, but note that this will slightly increase the amount of memory consumed by the `aws-node` pod.

**Comment by jdn5126** [comment]
Closing as this merged in #2567 and will ship in v1.15.1
